service/metrics/rpc.go

```diff
@@ -126,6 +130,98 @@ func (rpc *rpcServer) Observe(m *Metric, ok *bool) (err error) {
 		return fmt.Errorf("collector `%s` does not support method `Observe`", m.Name)
 	}
 
+	// RPC, set ok to true as return value. Need by rpc.Call reply argument
+	*ok = true
+	return nil
+}
+// Register is used to register new collector in prometheus
+// THE TYPES ARE:
+// 	NamedCollector -> Collector with the name
+// 	bool -> RPC reply value
+// RETURNS:
+// 	error
+func (rpc *rpcServer) Register(c *NamedCollector, ok *bool) (err error) {
```

Comment:
Can you rename it to `Declare` as convention for soft inserts?

Reply:
Sure